src/utils.js

```diff
@@ -332,14 +351,14 @@ function getTopItems(items, top_k = 0) {
     // if top == 0, return all
 
     items = Array.from(items)
-        .map((x, i) => [i, x])            // Get indices ([index, score])
-        .sort((a, b) => b[1] - a[1])      // Sort by log probabilities
+        .map((x, i) => [i, x]) // Get indices ([index, score])
+        .sort((a, b) => b[1] - a[1]); // Sort by log probabilities
```

Comment:
Is the linter able to leave this alone? Aligned comments make it more readable and easier to edit using multi-cursors.